website/docs/zh/blog/announcing-1-7.mdx

```diff
@@ -0,0 +1,389 @@
+---
+date: 2025-12-29 16:00:00
+sidebar: false
+---
+
+_2025 年 12 月 29 日_
+
+# Rspack 1.7 发布公告
+
+![Rspack 1.7](https://assets.rspack.rs/rspack/rspack-banner-v1-7.png)
+
+---
+
+我们很高兴地宣布 Rspack 1.7 已正式发布！
+
+值得关注的变更如下：
+
+- 新特性
+  - [SWC Wasm 插件兼容性提高](#swc-wasm-插件兼容性提高)
+  - [Lazy Compilation 默认开启](#lazy-compilation-默认开启)
+  - [支持 Import Bytes](#支持-import-bytes)
+  - [实验特性稳定化](#实验特性稳定化)
+- Rstack 进展
+  - [Rsbuild](#rsbuild)
+  - [Rspress](#rspress)
+  - [Rslib](#rslib)
+  - [Rstest](#rstest)
+  - [Rsdoctor](#rsdoctor)
+
+## 新特性
+
+### SWC Wasm 插件兼容性提高
+
+为了提升 Rspack 中 SWC Wasm 插件的兼容性，我们在这一版本中升级到了最新的 SWC 依赖。这次升级不仅带来了性能提升和 bug 修复，还在 [Wasm 插件兼容性方面的做出重要改进](https://swc.rs/docs/plugin/ecmascript/compatibility)。
+
+在之前的实现中，SWC 的 Wasm 插件与不同版本的 `@swc/core` 存在兼容性问题：由于 AST 序列化/反序列化机制依赖于版本一致的内部数据结构，旧版本插件在 host 更新后往往无法正常工作，给插件维护者和使用者带来了较大的升级和维护负担。
+为了解决这个痛点，SWC 引入了一系列兼容性改进，包括：
+
+- 使用**自描述式的 [cbor](https://www.rfc-editor.org/rfc/rfc8949.html) 序列化**方案取代原先对版本敏感的 [rkyv](https://rkyv.org/)，使 Wasm 插件能更好地适应 AST 结构的变更；
+- 为 AST 中的枚举类型增加 `Unknown` 变体，从而在遇到新字段或新节点时提升容错能力。
+
+这些改动极大改善了 Wasm 插件在不同 SWC 版本间的兼容性，降低了插件的维护成本，也让 Rspack 的生态插件能够更稳定地协同工作。在此基础上，官方现有的大部分 SWC 插件已经适配了这些兼容性机制，并且我们也正在积极与社区合作，帮助更多插件提升兼容性支持，让 Rspack 在插件生态方面有更平滑的体验。
+
+如果你是插件作者，请参阅 [Compatibility for wasm plugin](https://swc.rs/docs/plugin/ecmascript/compatibility#make-your-plugin-compatible) 文档提升你的插件的兼容性。
+
+### Lazy Compilation 默认开启
+
+Rspack 的 [LazyCompilation](/config/lazy-compilation) 特性在 1.5.0 发布时已经稳定。通过以下配置就能默认开启。
+
+```js title="rspack.config.mjs"
+export default {
+  lazyCompilation: true,
```

Comment:
@stormslowly we should mention that only `lazyCompilation.imports` is enabled by default, not include `lazyCompilation.entries`.